### ISSUE TITLE
Question about input preprocessing

### ISSUE THREAD

**Issue by kooweat** [dalgu90/resnet-18-tensorflow#13]
Hello,

Thank you for your contribution, I really appreciate it.
According to your code, you normalized input using mean and std of the imagenet as,
imagenet_mean = np.array([0.485, 0.456, 0.406], dtype=np.float32) * 255.0
imagenet_std = np.array([0.229, 0.224, 0.225], dtype=np.float32) * 255.0
image = (image - imagenet_mean) / imagenet_std

However, it seems little different from the official torch version normalization method used for resnet training.
Which one is the correct one that you used for the pretraining? (torch version? or the one in your code?)

Thanks in advance.
